src/database/utilities/createRecord.js

```diff
@@ -145,7 +145,7 @@ const createRequisition = (
   const { name: orderTypeName, maxMOS, thresholdMOS } = orderType;
   const regimenData =
     program && program.parsedProgramSettings ? program.parsedProgramSettings.regimenData : null;
-  const daysToSupply = monthsLeadTime ? (monthsLeadTime + maxMOS) * 30 : 30;
+  const daysToSupply = monthsLeadTime + (maxMOS || 1) * 30;
```

Comment:
I think there's an associativity bug here. Think it should be:

```
const daysToSupply = (monthsLeadTime + (maxMOS || 1)) * 30;
```

Reply:
One last small thing which may be a stupid question... should we assume that if `maxMOS` is `NaN` or similar, that it should be `1`? Is it possible that `maxMOS` should ever be `0` or fractional?

Reply:
cbb01c806e6913dd641d808fc9688b9322461462

Reply:
I think it is fine to assume maxMOS being 0 is an error state and can be defaulted to 1. Fractions will be chosen before the default, still